packages/graphql-cli-codegen-plugin/README.md

```diff
@@ -1,9 +0,0 @@
-# GraphQL Code Generator plugin for GraphQL CLI
```

Comment:
let's keep this readme in this directory and just have a message that this is no longer maintained and we link to the new docs you added